lib/excon.rb

```diff
@@ -116,7 +116,7 @@ def new(url, params = {})
         :port       => uri.port,
         :query      => uri.query,
         :scheme     => uri.scheme
-      }.merge!(params)
```

Comment:
I think it was intended as a means of avoiding having to allocate memory over again for this? It seems like it shouldn't hurt anything either way though. What do you think?